bundles/org.openhab.core/src/main/java/org/openhab/core/common/CronAdjuster.java

```diff
@@ -10,7 +10,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.openhab.core.internal.scheduler;
+package org.openhab.core.common;
```

Comment:
Imo `org.openhab.core.scheduler` is the better place to move this class. Isn it?

Reply:
Good idea!